webapp/src/test/resources/application-test.properties

```diff
@@ -17,4 +17,6 @@ l10n.file-system-drop-exporter.base-path=target/test-output/fileSystemDropExport
 
 l10n.repositoryStatistics.computeOutOfSla=true
 
-l10n.branchNotification.noop.enabled=true
\ No newline at end of file
+l10n.branchNotification.noop.enabled=true
+
+l10n.thirdPartySyncJobs.testRepository.thirdPartyProjectId=testProjectId
```

Comment:
Since this properties is added here, the test now gets an additional entry (the 2 entries from the local test configuration + this additional new entry). So I guess one way to fix the test is to add the entry in the expected result. This is is not super clean/isolated but I'd have to dig more to find a clean solution. If you update the test as mentioned above, can you add a comment saying the new entry comes from the "application-test.properties"?